sim/src/events.rs

```diff
@@ -59,7 +59,8 @@ pub enum Event {
     },
     TripCancelled(TripID),
     TripPhaseStarting(TripID, PersonID, Option<PathRequest>, TripPhaseType),
-
+    TripIntersectionDelay(TripID, TurnID, u8),
+    LaneSpeedPercentage(TripID, LaneID, u8),
```

Comment:
Similarly, I think having the actual speed and max speed here and doing the division later

Reply:
So you would move the threshold logic inside the event handling as well?

Reply:
Done